pkgs/applications/networking/instant-messengers/pidgin-plugins/purple-slack/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, pidgin }:
+
+stdenv.mkDerivation {
+  pname = "purple-slack";
+  version = "unstable-2019-08-18";
+
+  src = fetchFromGitHub {
+    owner = "dylex";
+    repo = "slack-libpurple";
+    rev = "be97802c7fd0b611722d2f551756e2a2672f6084";
+    sha256 = "0l6hwnnv6zzszxkk0a3nli88w5gijvlc9qgkkai0sb4v4c504y5v";
+  };
+
```

Comment:
```suggestion
  nativeBulidInputs = [ 
    pkgconfig
  ];
```

^ above too

Reply:
Should be resolved in latest force push :)